### ISSUE TITLE
[SXDL] UserWarning: None of the inputs have requires_grad=True. Gradients will be None

### ISSUE THREAD

**Issue by entrpn** [kohya-ss/sd-scripts#822]
I am finetuning a model and it generally works fine, but when I set gradient_accumulation_steps = 16, I get the error:

UserWarning: None of the inputs have requires_grad=True. Gradients will be None

This is the command:

```bash
nohup python sdxl_train.py --sample_prompts='/sample_prompt.txt' \
--dataset_config='dataset_config.toml' \
--config_file='config_file.toml' \
--save_every_n_steps 2958 \
--no_half_vae \
--xformers --full_fp16 \
--noise_offset 0.0357 \
--scale_v_pred_loss_like_noise_pred \
--v_parameterization &
```

This works fine with lower accumulation steps (4, 8), but 16 always results in this error. My lr = 4e-7.
